src/python/pants/option/global_options.py

```diff
@@ -397,6 +397,20 @@ def register_bootstrap_options(cls, register):
             help="Read additional specs from this file (e.g. target addresses or file names). "
             "Each spec should be one per line.",
         )
+
+        register(
+            "--query",
+            type=list,
+            default=[],
+            fromfile=True,
+            metavar="<query-expr>",
+            # TODO: rename this to restart_daemon=False! It's not clear where this option is
+            # even consumed.
+            daemon=False,
```

Comment:
You can remove this parameter: it was inverted in #10035, so `daemon=False` is now the default.